test/controllers/build_commands_controller_test.rb

```diff
@@ -48,6 +49,33 @@
           project.reload.build_command.must_equal nil
         end
       end
+
+      # It isn't possible through the UI to associate a build command with multiple projects.
+      # However, it is possible through the API and the console to do this. So in the case that
+      # this does happen, we should prevent such commands from being destroyed.
```

Comment:
@grosser That would be through `:stage_commands`. I don't think it's possible through the UI to make more than one Project have the same `build_command_id`.

```
class Command
  has_many :projects, foreign_key: :build_command_id
end
```